iodhbwm.tex

```diff
@@ -261,13 +261,16 @@
                 Eine ausführliche Beschreibung befindet sich auf \url{https://tex.stackexchange.com/a/69484/142408}.
             
             \subsubsection{TeXlive}
-                \Warning{Die Installationsanweisung für TeXlive wurde nicht getestet!}
+                \Info{Manche dieser Schritte können villeicht übersprungen werden!}
```

Comment:
The documentation is set with [skdoc](https://ctan.org/pkg/skdoc?lang=de). There is only a `\Notice` command.